00-programming-fundamentals/01-how-to-osx.md

```diff
@@ -1,55 +1,150 @@
-- `⌘t`: open a new tab (includes file search in text editors)
-- `⌘r`: reload page (Browsers only)
-- `⌘,`: open settings pane
-- `⌘/`: make selection or line a comment (text editors only)
-
-Hotkeys can be at the system level or specific to a certain application. Not all application will respond to these hotkeys, but the majority that we use (Terminal, major browsers, Atom, Sublime) will.
-
-It takes some practice but after you use these enough, they will become intuitive. Many of the hotkeys I use, I couldn't tell you what they are, my hands just know how to do them.
-Resources. Pick top 5 Hotkeys you'd like to use, and force yourself to use them.
-
-Hotkeys can be added or changed under `Keyboard` in the OS X _System Preferences_.
+
+These keyboard shortcuts work with a variety of applications to let you navigate within an application or between applications.  
+
+Hot Key Combo | Mac Symbols | What it does
+--------|:----------:|------------------
+Command-w | `⌘w`     | Close the Current tab.
+Command-shift-w | `⌘⇧w`     | Close the Current window (all the tabs).
+Command-q | `⌘q`     | Quit the application (`⌘⇧w` keeps the application running).
+Command-tab | `⌘⇥`   | Switch between applications.  Continue to hold the `⌘` button to display all open programs each tap on `⇥` will switch to the next app.  You can use `⌘⇧⇥` to switch to the previous app.
+Command-backtick | ⌘ backtick | Switch between windows in the same application.  
+Command-Space | `⌘␣` | Turn on spotlight.  Spotlight lets you search applications, files and do basic functions like math & conversions.
+Command-space-leftbracket | `⌘⇧[` | Move one tab to the left.
+Command-space-rightbracket | `⌘⇧[` | Move one tab to the right.
+Command-Shift-t | `⌘⇧t` | open the last closed item.
+Command-f  | `⌘f`  | Find.  Opens a panel to search for a term.
+Command-shift-f | `⌘⇧f` | Find all.  Searches all files (text editors like Atom & Sublime only).
+Command-n | `⌘n` | Opens a new window (or file depending on the application).
+Command-t | `⌘t` | Opens a new tab in a browser
+Command-r | `⌘r` | Reloads the page in a browser
+Command-comma | `⌘,` | Opens an app's preferences page. 
+Command-/ | `⌘/` | Make either the current line or a selection of text a comment (Sublime or Atom only)
+Command-shift-f | `⌘⇧f` | Fullscreen
+
+
+## A Word About Mice
+
+Steve Jobs had a [weird thing](http://www.cultofmac.com/269222/steve-jobs-hated-idea-multi-button-mouse-designer-claims/) about single-button mice.  Trust me, it's bizarre.  Over time Apple's added 2-button mouse functionality to MacOS, mostly to open rarely accessed sub-menus, and you'll be surprised how little you actually use it, if you think about it at all.  You can make a right click on your laptop trackpad by either by clicking and holding the option key down, or pressing with two fingers on the trackpad.  Or... you can just plug in a regular PC mouse and it'll work fine.  
+
+
+## Using a Windows Keyboard
```

Comment:
I'm not sure if we need to include this section - i wouldn't recommend this for someone who is switching to a mac since i think it will be more confusing